### PR TITLE
BETA: Register sajidhussain.is-a.dev

### PR DESCRIPTION
**Pull request by Sajid-tech**
Added `sajidhussain.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by Sajid-tech** [comment]
i have updated my json , pls reveiw it

File: domains/sajidportfolio.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Sajid-tech",
+        "email": "s.khan9430319425@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://sajid-tech.github.io/sajidhussain.github.io"
```
Comment (21Z) [inline domains/sajidportfolio.json:7]: `CNAME` can't contain protocol.
`CNAME` can't contain `/`
(tip: rename your github website repo to `sajid-tech.github.io`)

File: domains/sajidhussain.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "repo": "https://github.com/Sajid-tech/sajidhussain.github.io",
+    "owner": {
+        "username": "Sajid-tech",
+        "email": "s.khan9430319425@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://sajid-tech.github.io/sajidhussain.github.io"
```
Comment (21Z) [inline domains/sajidhussain.json:8]: `CNAME` can't contain protocol.
`CNAME` can't contain `/`
(tip: rename your github website repo to `sajid-tech.github.io`)

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/sajidhussain.json (hunk 2 above)
Comment (Sajid-tech) [reply domains/sajidhussain.json]: okay I have updated my JSON file as per your instruction, pls look into it

**Review by Sajid-tech — COMMENTED** [review]
okay , i have rename my github website name

File: domains/sajidhussain.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "repo": "https://github.com/Sajid-tech/sajid-tech.github.io",
+    "owner": {
+        "username": "Sajid-tech",
+        "email": "s.khan9430319425@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://sajid-tech.github.io"
```
Comment (21Z) [inline domains/sajidhussain.json:8]: `CNAME` should not contain protocol
```suggestion
        "CNAME": "sajid-tech.github.io"
```

File: domains/sajidportfolio.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Sajid-tech",
+        "email": "s.khan9430319425@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://sajid-tech.github.io"
```
Comment (21Z) [inline domains/sajidportfolio.json:7]: `CNAME` should not contain protocol
```suggestion
        "CNAME": "sajid-tech.github.io"
```

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/sajidhussain.json (hunk 3 above)
Comment (Sajid-tech) [reply domains/sajidhussain.json]: okay updated thej json

File: domains/sajidportfolio.json (hunk 4 above)
Comment (Sajid-tech) [reply domains/sajidportfolio.json]: updated

**Review by Sajid-tech — COMMENTED** [review]
updated

**Pull request closed without merging**